querybook/webapp/ui/Comment/Comments.tsx

```diff
@@ -85,66 +90,97 @@ export const Comments: React.FunctionComponent<IProps> = ({
 
     const handleEditComment = React.useCallback(
         (
-            commentId: number,
-            commentText: DraftJs.ContentState,
-            parentCommentId?: number
+            commentId: number = null,
+            commentText: DraftJs.ContentState = emptyCommentValue,
+            parentCommentId: number = null
         ) => {
             setEditingCommentId(commentId);
-            setEditingCommentParentId(parentCommentId || null);
             setCurrentComment(commentText);
+            setEditingCommentParentId(parentCommentId);
+            if (parentCommentId && !openThreadIds.has(parentCommentId)) {
+                setOpenThreadIds((ids) => new Set(ids).add(parentCommentId));
+            }
         },
-        []
+        [openThreadIds]
     );
 
-    const handleCommentSave = React.useCallback(() => {
-        if (editingCommentId) {
-            editComment(editingCommentId, currentComment);
+    const handleCommentSave = React.useCallback(
+        (values: { text: DraftJs.ContentState }, { resetForm }) => {
```

Comment:
is it possible to add async/await so that the reset of submitted comments only happen when the editComment or handleCreateComment is finished?